scripts/generate.sh

```diff
@@ -268,6 +268,24 @@ rust () {
   fi
 }
 
+elixir () {
+  echo "Generating Elixir..."
+  
+  dir="clients/${PROJECT}/elixir"
+  rm -rf "$dir" || true
+  mkdir -p "$dir"
+
+  openapi-generator-cli generate -i "${SPEC_FILE}" \
+    	-g elixir \
+	    -o "$dir" \
+	    --git-user-id ory \
+	    --git-repo-id sdk \
+	    --git-host github.com \
+	    -c ./config/client/elixir.yml.proc.yml
+
+  cp "LICENSE" "clients/${PROJECT}/elixir"
```

Comment:
We do this now in `to_git` in the release step :)

```suggestion
```

Reply:
Removed.

The rebase for this branch is a bit huge so currently avoiding it - let me know if you require a rebase here